programs/bpf_loader/src/syscalls/cpi.rs

```diff
@@ -733,41 +739,89 @@ where
     Ok(accounts)
 }
 
+#[cfg(test)]
+static_assertions::const_assert_eq!(ACCOUNT_META_SIZE, 34);
+const ACCOUNT_META_SIZE: usize = size_of::<AccountMeta>();
+
 fn check_instruction_size(
     num_accounts: usize,
     data_len: usize,
     invoke_context: &mut InvokeContext,
 ) -> Result<(), EbpfError<BpfError>> {
-    let size = num_accounts
-        .saturating_mul(size_of::<AccountMeta>())
-        .saturating_add(data_len);
-    let max_size = invoke_context.get_compute_budget().max_cpi_instruction_size;
-    if size > max_size {
-        return Err(SyscallError::InstructionTooLarge(size, max_size).into());
+    if invoke_context
+        .feature_set
+        .is_active(&feature_set::loosen_cpi_size_restriction::ID)
```

Comment:
for consistency sake `loosen_cpi_size_restrictions::id()`

Reply:
Updated